docs/root/intro/arch_overview/upstream/aggregate_cluster.rst

```diff
@@ -0,0 +1,112 @@
+| 71%                   | 1%                    | 0%                    | 100%                  | 100%                  | 100%               | 0%                   |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 71%                   | 0%                    | 0%                    | 100%                  | 100%                  | 99%                | 1%                   |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 50%                   | 0%                    | 0%                    | 50%                   | 0%                    | 70%                | 30%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 20%                   | 20%                   | 10%                   | 25%                   | 25%                   | 70%                | 30%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 20%                   | 0%                    | 0%                    | 20%                   | 0%                    | 50%                | 50%                  |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 0%                    | 0%                    | 0%                    | 100%                  | 0%                    | 0%                 | 100%                 |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+| 0%                    | 0%                    | 0%                    | 72%                   | 0%                    | 0%                 | 100%                 |
++-----------------------+-----------------------+-----------------------+-----------------------+-----------------------+--------------------+----------------------+
+
+Note: The above load balancing uses default :ref:`overprovisioning factor <arch_overview_load_balancing_overprovisioning_factor>` which is 1.4.
+
+To sum this up in pseudo algorithms:
+
+::
+
+  health(P_X) = min(100, 1.4 * 100 * healthy_P_X_backends / total_P_X_backends), where total_P_X_backends is the number of backends for priority P_X after linearization
+  normalized_total_health = min(100, Σ(health(P_0)...health(P_X)))
+  priority_load(C_0) = min(100, Σ(health(P_0)...health(P_k)) * 100 / normalized_total_health), where P_0...P_k belong to C_0
```

Comment:
Why do you use `k` here and `x` below for similar variable? It's best to be consistent. Also, why count from 0 to `k` (which I assume is all the priorities) here but `x` to `X` below? Sorry to be nit picky, but I'm trying to put myself in the shoes of someone looking at this for the first time.

Reply:
Priorities are from 0 to X after linearization, in which 

```
P_0 ... P_k ... ...P_x ... P_X
^        ^         ^       ^
cluster 0          cluster X 
```

where `k` and `x` denote different priority

Reply:
Makes sense, can you add this nice ASCII art :) ?

Reply:
added